ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/BlockTransactionSelector.java

```diff
@@ -173,7 +173,7 @@ private TransactionSelectionResult evaluateTransaction(final Transaction transac
     }
 
     if (transactionTooLargeForBlock(transaction)) {
-      LOG.trace("{} too large to select for block creation", transaction);
+      LOG.info("{} too large to select for block creation", transaction);
```

Comment:
Is this an intended change? Doesn't seem to be mentioned in either the issue or the ticket

Reply:
This was indeed unintended and has been updated now, thanks.

Reply:
b9a5595